### DISCUSSION TITLE
Cant get it to work

### DISCUSSION THREAD

**Discussion by Neofeliz** [isid0re/SoloLeveling#189] [Q&A] [answered]
Good day, 
I just downloaded the script but when I try to run it I’m getting some errors and my character is standing still doing nothing. 
![8264CFD9-453B-4BA5-A8E0-93D696A82B73](https://user-images.githubusercontent.com/1080394/113519937-e16fe200-958f-11eb-920d-5e1b836bc03c.jpeg)

What am I doing wrong? It’s a fresh install of kolbot “newest version” please help.

**Comment by isid0re** [comment, marked as answer]
To confirm you are using the https://www.github.com/blizzhackers/kolbot version 

If yes then,

What are the D2BS profile name and info tag settings?

**Reply by Neofeliz** [reply]
Just reinstalled kolbot from your link and it works now, I tought I downloaded the version from the link from the main page. Think I did some thing wrong :) 
Thank for your time
